test/langtools/tools/javac/reflect/CodeReflectionTester.java

```diff
@@ -45,12 +46,19 @@ public static void main(String[] args) throws ReflectiveOperationException {
             System.exit(1);
         }
         Class<?> clazz = Class.forName(args[0]);
+        check(clazz);
+    }
+
+    public static void check(Class<?> clazz) throws ReflectiveOperationException {
         for (Method m : clazz.getDeclaredMethods()) {
             check(m);
         }
         for (Field f : clazz.getDeclaredFields()) {
             check(f);
         }
+        for (Class<?> c : clazz.getDeclaredClasses()) {
```

Comment:
Interestingly, I discovered that the harness was ignoring `CodeReflection` annotations in nested classes -- this is fixed now.